addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/handler/TeslaHandler.java

```diff
@@ -129,8 +131,10 @@
 
     protected boolean allowWakeUp = true;
     protected long lastTimeStamp;
-    protected long intervalTimestamp = 0;
-    protected int intervalErrors = 0;
+    protected long apiIntervalTimestamp = 0;
+    protected int apiIntervalErrors = 0;
+    protected long eventIntervalTimestamp = 0;
+    protected int eventIntervalErrors = 0;
```

Comment:
Can you remove the redundant 0 initializations? They also generate more bytecode. I.e. slightly more storage and slightly less performance, see [Java Practices](http://www.javapractices.com/topic/TopicAction.do?Id=14).

Reply:
@wborn done